thorlcr/thorutil/thormisc.hpp

```diff
@@ -409,6 +412,14 @@ inline void ActPrintLog(const CGraphElementBase *container, const char *format,
     ActPrintLogArgs(container, thorlog_ecl, MCdebugProgress, format, args);
     va_end(args);
 }
+inline void ActPrintLog(const CGraphElementBase *container, unsigned traceLevel, const char *format, ...) __attribute__((format(printf, 3, 4)));
```

Comment:
Is this function, and the GraphPrintLog function actually used?

Reply:
yes, both are used.